mbed_greentea/mbed_test_api.py

```diff
@@ -247,6 +250,9 @@ def get_binary_host_tests_dir(binary_path, level=2):
         # Example:
         # $ mbedhtrun -p :9600 -f "tests-mbed_drivers-generic_tests.bin" -m K64F --grm raas_client:10.2.203.31:8000
         cmd += ['--grm', global_resource_mgr]
+
+        if tags:
+            cmd+= ["--allocate-on-tag", str(tags)]
```

Comment:
Please add a space in front of the `+=` operator.

Reply:
Further, does `str(tags)` actually correctly format the tags here?

Reply:
Is there another format that would work better? I haven't had issues parsing the string in the htrun layer once it's passed on here with up to 3 tags specified, but there could be a better implementation.